README.md

```diff
@@ -1,34 +1,45 @@
-![logo](https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png)
-
-![CI](https://github.com/ManimCommunity/manim/workflows/CI/badge.svg)
-[![Documentation Status](https://readthedocs.org/projects/manimce/badge/?version=latest)](https://manimce.readthedocs.io/en/latest/?badge=latest)
-[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat)](http://choosealicense.com/licenses/mit/)
-[![Manim Subreddit](https://img.shields.io/reddit/subreddit-subscribers/manim.svg?color=ff4301&label=reddit)](https://www.reddit.com/r/manim/)
-[![Manim Discord](https://img.shields.io/discord/581738731934056449.svg?label=discord)](https://discord.gg/mMRrZQW)
-
-Manim is an animation engine for explanatory math videos. It's used to create precise animations programmatically, as seen in the videos at [3Blue1Brown](https://www.3blue1brown.com/).
-
-> NOTE: This repository is maintained by the Manim Community, and is not associated with Grant Sanderson or 3Blue1Brown in any way (though we are definitely indebted to him for providing his work to the world). If you want to study how Grant makes his videos, head over to his repository (3b1b/manim). This is a more frequently updated repository than that one, and is recommended if you want to use Manim for your own projects.
+<p align="center">
+    <img src="https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png" />
```

Comment:
[Non-Blocking]: In case that we centre this logo, I think it would be nice to rescale it a bit: to reduce the length and broaden the width, so it will look more like this: https://github.com/matplotlib/matplotlib

Reply:
We can't rescale it (it would look stretched). We have to crop it. So if anyone wants to crop it, go ahead, but I kinda like the whitespace 😃

Reply:
As a warning, this file must stay in this location and keep this name or it'll break the documentation for v0.1. So even if you do want a new picture here (an svg would be nice), you shouldn't change the old one.